src/error/specification

```diff
@@ -1195,3 +1195,10 @@ severity:error
 ingroup:plugin
 module:pluginprocess
 macro:PLUGIN_PROCESS_COMMAND
+
+number:192
+description:The mounted specification failed the typecheck
+severity:warning
```

Comment:
Why only warning? For writing it should be an error?

Reply:
the idea was that otherwise it would always trigger an error when setting some keys in the spec namespace with the typechecker mounted, as there are probably quite some cases that need several keys or metakeys until you get to a meaningful specification again. Or is there some kind of "transaction" so i can set several keys at once using kdb set and then commit the changes to the kdb all at once

Reply:
Yes, of course there is a transaction, you can modify a whole KeySet at once :wink:

Reply:
ok then it'll become an error

Reply:
for the read direction a warning is needed. Because otherwise users would not be able to fix problems already  in the spec (for any reason).